### PR TITLE
Send optional structure when post

### PR DESCRIPTION
**Pull request by jean553**

### CODE REVIEW

File: tests/test_create_sentence.rs [hunk 1]
```diff
@@ -39,6 +39,82 @@ fn test_post_sentence_returns_200() {
...
+        response.status(),
+        StatusCode::Created,
+    );
+
+    assert_eq!(
+        db::get_structure(
+            &connection,
+            &sentence_uuid,
+        ),
+        "".to_string(),
```
Comment (allan-simon) [inline tests/test_create_sentence.rs]: should be null in database

Comment (jean553) [reply tests/test_create_sentence.rs]: Does it make sense to return an Option<T> here from `get_structure` ?

File: tests/db.rs [hunk 2]
```diff
@@ -214,7 +214,12 @@ pub fn get_structure(
         .expect("0 results, expected one...")
     ;
 
-    row.get(0)
+    /* get_opt(_) returns an error wrapped into an Option
+       object if the row exists and the structure column is null */
+    match row.get_opt(0) {
+        Some(Ok(data)) => Some(data),
+        Some(Err(_)) | None => None,
```
Comment (allan-simon) [inline tests/db.rs]: if there's an Err , we should panic, no ?

Comment (allan-simon) [reply tests/db.rs]: or at least consider the test fail ?

Comment (jean553) [reply tests/db.rs]: That makes sense too. We could panic if there is an error (null column) or if the result is None. Ok ?

**Pull request merged**